### ISSUE TITLE
[BUG] Docker Desktop redux

### ISSUE THREAD

**Issue by justinkb** [Nevuly/WSL2-Rolling-Kernel-Issue#17]
Well this is awkward, but I made a mistake with the diagnosing of this issue. When I tried to see if CONFIG_MODULE_FORCE_{,UN}LOAD=y on top of 2c046a84e169 would fix it, I ended up using the wrong built kernel in .wslconfig and so incorrectly concluded that was the fix. I can now say that that doesn't fix the issue after all. So now I'm going back to figuring out the real issue

**Comment by Nevuly** [comment]
Moved to WSL2-Rolling-Kernel-Issue repository to avoid referencing issue when I try to merge kernel commits.

**Comment by Nevuly** [comment]
https://github.com/Nevuly/WSL2-Linux-Kernel-Rolling/commit/b22a99be4502fe7da02fcdd3ba7d4ff859f8193e
https://github.com/Nevuly/WSL2-Linux-Kernel-Rolling/commit/70d9fe25df1814ff62e72b9bac2eab025fb5bc94
https://github.com/Nevuly/WSL2-Linux-Kernel-Rolling/commit/f686749b9b61951af2500b34b14e5a89dc4c13a7

I regenerated defconfigs.

 - Based defconfigs from `linux-msft-wsl-5.15.y`
 - Bringup some feature configs from `linux-msft-wsl-6.6.y`
 - Re-enable netfilter configs for support WireGuard VPN

I checked in private build, and Docker Desktop work correctly.

Thanks.
